### DISCUSSION TITLE
[Docs]: integration with TinyMCE link

### DISCUSSION THREAD

**Discussion by bb-art** [remix-run/react-router#10959] [Q&A]
### Describe what's incorrect/missing in the documentation

With tinyMCE-react (or any other wysiwyg editor) I can create html content, including hyperlinks, which are plain <a> elements. Is there a way to connect these hyperlinks to react-router? 
Afaik there is no react router Link component pugin available for the editor.

**Comment by adtm** [comment]
A bit random, but maybe you have an example how to integrate with `remix` and `tinymce` in general? Hitting a bit of a roadblock.
